src/pages/index.vue

```diff
@@ -50,9 +50,7 @@ function navigateToCreatePool() {
     <BalStack vertical>
       <div class="px-4 xl:px-0">
         <div class="flex justify-between items-end mb-8">
-          <h3>
-            {{ $t('investmentPools') }}
-          </h3>
+          <h3>{{ networkConfig.chainName }} {{ $t('pools') }}</h3>
```

Comment:
Only issue with this is for Arbitrum it ends up being "Arbitrum One Pools". Maybe we can just change the chain name in the config? I don't think there should be any issue caused by that.

Reply:
Updated